main.py

```diff
@@ -1,10 +1,56 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    Race.objects.all().delete()
+    Skill.objects.all().delete()
+    Guild.objects.all().delete()
+    Player.objects.all().delete()
+
+    with open("players.json") as f:
+        players = json.load(f)
+        for player in players:
+            race_name = players[player]["race"]["name"]
+            race_description = players[player]["race"]["description"]
+            race, create = Race.objects.get_or_create(
```

Comment:
You could omit create variable since it is not used in the code anywhere else
```suggestion
            race, _ = Race.objects.get_or_create(
```
Add this change in all relevant places